src/types/SpliceInfo.ts

```diff
@@ -1,18 +1,20 @@
 import * as utils from '../utils';
 
 export type SpliceTypes = 'OUT' | 'IN' | 'RAW';
-export type SpliceTagNames =
-    | 'EXT-X-CUE-OUT-CONT'
-    | 'EXT-X-CUE-OUT-CONT'
-    | 'EXT-OATCLS-SCTE35'
-    | 'EXT-X-ASSET'
-    | 'EXT-X-SCTE35'
-    | 'EXT-X-CUE';
+export type SpliceTagNames = 'EXT-X-CUE-OUT-CONT' | 'EXT-OATCLS-SCTE35' | 'EXT-X-ASSET' | 'EXT-X-SCTE35' | 'EXT-X-CUE';
+
+export interface TransmitSpliceValue {
```

Comment:
Not sure I like having this in here, but it's a private repo so making it Flo specific ain't so bad 🤷‍♀️

Reply:
I'd argue the Flo-specific stuff was added a while ago. This is just giving it stronger typing